src/components/preferences/HashtagList.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Redirect } from 'react-router-dom';
+
+import { addHashtag } from '../../actions';
+import { ListGroup, ListGroupItem, Badge } from 'reactstrap';
+
+import HashtagForm from './HashtagForm'
+
+class HashtagList extends React.Component {
+
+  renderHashtags () {
+    return this.props.hashtags.map((el, index) => <li key={index}>{el}</li>)
+  };
+
+  handleSubmit = (formData) => {
+    const { hashtags } = formData;
+    this.props.addHashtag('#' + hashtags);
+
+    var body = {
+      'update': {
+        'be_like': []
+      },
+      'add': {
+        'like_tags': [
+          hashtags
+        ]
+      }
+    };
+
+    fetch("https://private-cb530a-bravakin.apiary-mock.com/me", {
```

Comment:
Create a new file in /src/config and extract this URL to it, it will be easier to edit when the time comes (you can base yourself on what was done with the /src/private/conf.js)
Remember to choose descriptive names for the variables